src/utils/dom/getParent.js

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default element => {
+  const result = element.parentNode;
+
+  if (result) {
+    return result;
+  }
+
+  return null;
```

Comment:
Is this necessary? If the element doesn't have a parent, `element.parentNode` should be `null` as far as I understand, so it seems like you just do `return element.parentNode`. If there is a reason for this extra logic, a comment might help clarify for future developers.

Reply:
Nope, it is not necessary I think I was just carried away. The initial intent was to check if element is `null`, but then I dropped that idea.

Fixed.